graphistry/tests/test_embed_utils.py

```diff
@@ -10,42 +11,62 @@
 logger = logging.getLogger(__name__)
 
 dep_flag, _, _, _, _, _, _, _ = lazy_embed_import_dep()
+def check_cudf():
+    try:
+        import cudf
+        return True, cudf
+    except:
+        return False, None
 
-edf = pd.DataFrame([[0, 1, 0], [1, 2, 0], [2, 0, 1]],
-        columns=['src', 'dst', 'rel']
-)
-ndf_no_ids = pd.DataFrame([['a'], ['a'], ['b']], columns=['feat'])
-ndf_with_ids = pd.DataFrame([[0, 'a'], [1, 'a'], [2, 'b']],
-        columns = ['id', 'feat1']
-)
 
-graph_no_feat = graphistry.edges(edf, 'src', 'dst')
-graph_with_feat_no_ids = graph_no_feat.nodes(ndf_no_ids)
-graph_with_feat_with_ids = graph_no_feat.nodes(ndf_with_ids, 'id')
-graphs = [('no_feat', graph_no_feat), ('with_feat_no_ids', graph_with_feat_no_ids), ('with_feat_with_ids', graph_with_feat_with_ids)]
-d = 4
+has_cudf, cudf = check_cudf()
 
-kwargs = {'n_topics': 6, 'cardinality_threshold':10, 'epochs': 1, 'sample_size':10, 'num_steps':10}
+TEST_CUDF = False
+if "TEST_CUDF" in os.environ and os.environ["TEST_CUDF"] == "1":
+    TEST_CUDF = True
```

Comment:
so maybe should be 

```
# enable tests if has cudf and env didn't explicitly disable
is_test_cudf = has_cudf and os.environ["TEST_CUDF"] !== "0"

...

@pytest.mark.skipif(not is_test_cudf, ...
```

Reply:
(Not sure what we do elsewhere)

Reply:
shouldn't it be `os.environ["TEST_CUDF"] !== "1"` rather `os.environ["TEST_CUDF"] !== "0"`